### PR TITLE
Little polishing

### PR DESCRIPTION
**Pull request by hzlmn**
- Fixed import to utils on `conftest` (because of this test command was broken)
- Fixed typo in path to config

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/Makefile [hunk 1]
```diff
@@ -19,10 +19,10 @@ bash:
 # Linters & tests
 
 mypy:
-	@docker-compose run --rm $(PROJECT_NAME)_app mypy $(PROJECT_NAME)
+	mypy $(PROJECT_NAME)
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: i'm not sure that it is good idea. U need have `mypy` locally.

Comment (hzlmn) [reply create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: You mostly will have it on your system and it is not problematic to install it. But after thinking more I guess you are right and it is easy to tune make command if you need so i will revert it.

Comment (Arfey) [reply create_aio_app/template/{{cookiecutter.project_name}}/Makefile]: yes, thx

**Comment by Arfey** [comment]
thx 😀

**Pull request merged**